manual/cypher/cypher-docs/src/docs/dev/ql/functions/index.asciidoc

```diff
@@ -283,3 +283,24 @@ include::split.asciidoc[leveloffset=+1]
 include::reverse.asciidoc[leveloffset=+1]
 
 include::tostring.asciidoc[leveloffset=+1]
+
+ifndef::asciidoctor[:leveloffset: 1]
+
+[[query-functions-udf]]
+== User-defined functions ==
+User-defined functions are written in Java, deployed into the database and are called in the same way as any other Cypher function.
```

Comment:
If possible I don't want to mention procedures at all here, since this is a section about procedures. @jjaderberg I am not totally clear what you are suggesting, should I remove the sentence, remove the code snippet, or both?